garage/contrib/ros/envs/sawyer/launchers/trpo_real_sawyer_push.py

```diff
@@ -22,22 +28,26 @@
 def run_task(*_):
     initial_goal = np.array([0.6, -0.1, 0.80])
 
+    # Initialize moveit_commander
+    moveit_commander.roscpp_initialize(sys.argv)
```

Comment:
Can this be done from inside the SawyerEnv instead?

Reply:
it needs to be called before rospy.init_node, we put ropy.init_node in launcher file for some reasons before. I will take a look later.